src/index.js

```diff
@@ -0,0 +1,24 @@
+import { setFailed } from "@actions/core";
+import { processAnnotations } from "./annotate";
+
+// This is the entrypoint of the javascript app that processes the json files
+// from steampipe check to process the annotations.
+//
+// This app is executed with the list of checks from the input of the action
+// as separate arguments in the argument vector
+async function run() {
+  try {
+    let token = process.env["GITHUB_TOKEN"];
```

Comment:
```suggestion
    const token = process.env["GITHUB_TOKEN"];
```

If this value isn't changing, we should use `const` (low risk but better to make the value immutable)